Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/utils/Injection.kt

```diff
@@ -0,0 +1,10 @@
+package com.mentalmachines.droidcon_boston.utils
+
+import com.google.gson.Gson
+
+class Injection {
```

Comment:
this isn't really injection..it's more a service locator as JW would say ;-)

Reply:
I can put the icons back in. I just wanted to experiment a little 😅 
About the Injection class, I know it is Service locator, needed to isolate the GSON dependency so named it.

Reply:
experimenting is fair...was just voicing an opinion ;-)